core/src/main/java/org/openstack4j/model/common/TimeEntity.java

```diff
@@ -0,0 +1,26 @@
+package org.openstack4j.model.common;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ *  A simple entity which supports time data
+ * 
+ * @author bboyHan
+ */
+public interface TimeEntity {
+
+    /**
+     * created time
+     * 
+     * @return created time
+     */
+    String getCreatedTime();
```

Comment:
Do not return time as `String`. Rest of the codebase is using `Date`.

Reply:
got it